.github/workflows/cve-dependency-scan.yml

```diff
@@ -39,6 +40,7 @@ jobs:
       - name: Send link to slack
         id: slack
         uses: slackapi/slack-github-action@v1.23.0
+        if: always() && steps.dependencyCheckAnalyze.outcome == 'failure'
```

Comment:
Not sure `always()` is required since the steps are not in a separate job.

Let's simplify and drop the `continue-on-error` and use `failure()` instead: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-status-check-functions

Reply:
I included the `always()` because it seems when using `if` a default status check of `success()` is applied unless you include a different function: https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions

I'll remove `continue-on-error` and try out `failure()`